index.js

```diff
@@ -253,14 +253,19 @@ module.exports = {
   },
 
   _getEmberModulesAPIPolyfill(config) {
+    const blacklist = { '@ember/debug': ['assert', 'deprecate', 'warn'] };
```

Comment:
Can you extract the logic here into a `_getEmberModulesAPIBlacklist()` method?

Reply:
Addressed, was that what you had in mind?

Reply:
Ya, thank you